### PR TITLE
Appveyor Fixes

### PR DESCRIPTION
**Pull request by markbates**

### CODE REVIEW

File: appveyor.yml [hunk 1]
```diff
@@ -8,40 +8,25 @@ environment:
...
-    go build -v -tags sqlite github.com/gobuffalo/buffalo/buffalo
+    set GO111MODULE=on
 
-    go install -tags sqlite github.com/gobuffalo/buffalo/buffalo
+    go build -v github.com/gobuffalo/buffalo/buffalo
 
 test_script:
-- cmd: for /f "" %%G in ('go list github.com/gobuffalo/buffalo/... ^| find /i /v "/vendor/"') do ( go test -tags "sqlite integration_test" %%G & IF ERRORLEVEL == 1 EXIT 1)
+  - set GO111MODULE=on
+  - go test -race ./...
```
Comment (stanislas-m) [inline appveyor.yml]: Can we keep the integration tests? That's one of the reasons we have a specific build for Windows.

**Comment by markbates** [comment]
Yes! Sorry I missed that.

**Comment by markbates** [comment]
@stanislas-m added the integration tests back. sorry about that. :)

**Comment by stanislas-m** [comment]
@markbates Np. :)

**Pull request merged**